catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,170 @@
+local id = 'OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0';
+local subdir = 'OpenET';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'OpenET CONUS Ensemble Monthly Evapotranspiration v2.0',
+  version: '2.0',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The OpenET dataset includes satellite-based data on the total amount of water that
+    is transferred from the land surface to the atmosphere through the process of
+    evapotranspiration (ET). OpenET provides ET data from multiple satellite-driven models,
+    and also calculates a single "ensemble value" from the model ensemble. The models
+    currently included in the OpenET model ensemble are ALEXI/DisALEXI, eeMETRIC, geeSEBAL,
+    PT-JPL, SIMS, and SSEBop. The OpenET ensemble ET value is calculated as the mean of the
+    ensemble after filtering and removing outliers using the median absolute
+    deviation approach.  
+    All models currently use Landsat satellite data to produce ET data at a pixel size of
```

Comment:
Please insert a blank line to make a new paragraph.  Or if you meant it to still be a single paragraph, please reflow the paragraph.